docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -399,7 +399,16 @@ Enabling S3 Select pushdown
 You can enable S3 Select Pushdown using the ``s3_select_pushdown_enabled``
 Hive session property, or using the ``hive.s3select-pushdown.enabled``
 configuration property. The session property overrides the config
-property, allowing you enable or disable on a per-query basis.
+property, allowing you enable or disable on a per-query basis. Non-filtering
+queries (``SELECT * FROM table``) are not pushed down to S3 Select,
+as they are retrieving the entire object content.
+
+For uncompressed files, Scan Range feature of S3 Select is used.
+An Amazon S3 Select scan range request runs across the specified byte range.
+This range is aligned with the internal Hive splits for the query fragments
+that get pushed down to Select. Changes in the Hive connector performance
+tuning configuration properties would be reflected here as well.
```

Comment:
```suggestion
For uncompressed files, S3 Select scans ranges of bytes in parallel. The scan range
request runs across the byte range of the internal Hive splits for the query fragments
pushed down to S3 Select. Changes to the Hive catalog's :ref:`performance tuning
configuration properties <hive-performance-tuning-configuration>` are reflected
here as well.
```

Reply:
To make the anchor link work, please add
`.. _hive-performance-tuning-configuration:`
above line 734 of the hive connector page (/connector/hive.rst)

Reply:
Thanks a lot! Updated 👍

Reply:
Can't reply to the above comment. There are no changes needed for "Is S3 Select a good fit for my workload" section.

Reply:
It looks like only the ref link was added, can you apply the rest of the suggested edits?

Should look like the following:

```
For uncompressed files, S3 Select scans ranges of bytes in parallel. The scan range
requests run across the byte ranges of the internal Hive splits for the query fragments
pushed down to S3 Select. Changes to the Hive catalog's :ref:`performance tuning
configuration properties <hive-performance-tuning-configuration>` are reflected
here as well.
```

Reply:
This was rephrased a bit on our side as well. From a technical perspective, I think we should say Hive connector, not Hive catalog, as this is related to how the connector works. 

Does this work for you:

```
For uncompressed files, S3 Select scans ranges of bytes in parallel. The scan range
requests run across the byte ranges of the internal Hive splits for the query fragments
pushed down to S3 Select. Changes in the Hive connector :ref:`performance tuning
configuration properties <hive-performance-tuning-configuration>` are likely to impact
S3 Select pushdown performance.
```

Reply:
Yep, that makes sense to me.